phoenix-core/src/it/java/org/apache/phoenix/end2end/MaxLookbackIT.java

```diff
@@ -59,10 +57,31 @@
 @NeedsOwnMiniClusterTest
 public class MaxLookbackIT extends BaseUniqueNamesOwnClusterIT {
     private static final Log LOG = LogFactory.getLog(MaxLookbackIT.class);
-    private static final int MAX_LOOKBACK_AGE = 10;
+    private static final int MAX_LOOKBACK_AGE = 15;
     private static final int ROWS_POPULATED = 2;
+    public static final int WAIT_AFTER_TABLE_CREATION = 600000;
```

Comment:
Can we write the unit as part of const name?

Reply:
Suggest setting WAIT_AFTER_TABLE_CREATION = MAX_LOOKBACK_AGE * 100 ... just in case someone increases MAX_LOOKBACK_AGE and leaves WAIT_AFTER_TABLE_CREATION as is.